experimental/framework/tinylicious-client/src/TinyliciousClient.ts

```diff
@@ -82,6 +85,7 @@ export class TinyliciousClientInstance {
         containerId: string,
         containerRuntimeFactory: IRuntimeFactory,
         createNew: boolean,
+        iTelemetryBaseLogger? : ITelemetryBaseLogger,
```

Comment:
probably better to pass through the entire `TinyliciousContainerConfig` object now that it has multiple members instead of expanding it

Reply:
yeah good point